PLAT-3197 be kind to your colleagues (#42)
talis/talis.github.io: topics/ci.md

We use Continuous Integration[^1] for any code that we release.

> CI should be configured to trigger automatically on merges to the `master` branch, and should provision a clean environment. Branch builds should be confirgured we a `be nice to your colleagues` approval step to reduce unnecessary builds.

Reviewer: Not sure `be nice to your colleagues` makes sense out of context... also spelling/grammar

```suggestion
CI should be configured to trigger automatically on merges to the `master` branch, and should provision a clean environment. Branch builds should be configured with an approval step requiring manual intervention to reduce unnecessary builds.
```

Reviewer: It's worth noting that the step is actually called `be kind to your colleagues` currently anyway :D

Reviewer: Wonder if we should rename it anyway since it's purpose will soon be completely different - rather than preventing people from hogging limited resources, it will prevent people racking up bills

All this is to say, I think it's not a good idea to refer to the step by name in the handbook

Author: @camillef good point.
